site/how-it-works.md

```diff
@@ -70,12 +70,9 @@ Scope can also be used to feed reports to Weave Cloud. With Weave Cloud, you can
 
 To register with the service, go to [https://cloud.weave.works](https://cloud.weave.works), enter your email address and wait for the confirmation email that contains your service token.
 
-![`Weave Cloud` Signup](images/weave-cloud-signup.png)
+The cloud token is accessible from the Weave Cloud main page after you've logged in:
```

Comment:
This won't be true very soon, once we rework the landing page. Can you be unspecific about it? (or mention the Scope/Explore section)